jest.config.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
 // @ts-check
-const { getJestProjects } = require('@nx/jest');
+const { getJestProjectsAsync } = require('@nx/jest');
```

Comment:
I'm getting flashbacks to C#/.NET 😛

Reply:
The sync version is only deprecated and still there so sadly we have to have another name for this one!